odoo/addons/base/res/res_company_view.xml

```diff
@@ -116,46 +116,64 @@
         <record id="view_company_report_form" model="ir.ui.view">
             <field name="name">res.company.report.form</field>
             <field name="model">res.company</field>
+            <field name="inherit_id" ref="base.view_company_form" />
+            <field name="mode">primary</field>
             <field name="priority">1024</field>
             <field name="arch" type="xml">
-                <form string="Company">
-                    <field name="logo" invisible="1"/>
-                    <div class="alert alert-warning" attrs="{'invisible': [('logo','!=',False)]}">
-                        <p>
-                            <strong>No logo found!</strong>
-                            To get a beautiful document, you should review your
-                            company data: upload a logo, set
-                            your address, etc.
-                        </p>
-                        <button name="open_company_edit_report" type="object" string="⇒ Review Company Data" class="btn-link"/>
-                    </div>
+              <xpath expr="//sheet[@id='company_sheet']" position="before">
+                <div class="alert alert-warning" attrs="{'invisible': [('logo','!=',False)]}">
+                      To get a beautiful document, you should update your company informations.
+                </div>
+              </xpath>
+              <xpath expr="//h1[@id='company_title']" position="replace">
+                <h2>
+                    <field name="name"/>
+                </h2>
+              </xpath>
+                <field name="parent_id"  groups="base.group_multi_company" position="replace" invisible="1"/>
+              <xpath expr="//sheet[@id='company_sheet']" position="inside">
+                    <hr />
                     <div class="row mt16">
+                        <field name="external_report_layout" invisible="1" widget="text" />
                         <div class="col-xs-3 text-center">
-                            <button type="object" name="set_report_template" context="{'report_template': 'background'}">
+                            <button type="object" name="set_report_template" context="{'report_template': 'background'}" attrs="{'invisible': [('external_report_layout', '=', 'background')]}">
+                                <img src="/base/static/img/preview_background.png" class="img img-responsive img-thumbnail" alt="Preview Background Template"/>
+                            </button>
+                            <button class="btn-info" type="object" name="set_report_template" context="{'report_template': 'background'}" attrs="{'invisible': [('external_report_layout', '!=', 'background')]}">
```

Comment:
@d-fence  Any clue about the fact that the attrs aren't working here ?

Reply:
@tivisse I tried and it seems to work here ?!

Reply:
The only purpose of those attrs is to add/remove the blue line around the template button.
I tested locally against community and enterprise and it works as expected.

Reply:
Ok for me then.